lib/plug/upload.ex

```diff
@@ -161,7 +161,7 @@ defmodule Plug.Upload do
   defp i(integer), do: Integer.to_string(integer)
 
   defp path(prefix, tmp) do
-    {_mega, sec, mili} = :erlang.now
+    {_mega, sec, mili} = :os.timestamp
```

Comment:
I have added the scheduler_id to the path.
Also, I renamed "milis" to "micro" to be consistent with :os.timestamp/0 documentation.